ddl/partition.go

```diff
@@ -1983,13 +1983,17 @@ func (cns columnNameSlice) At(i int) string {
 	return cns[i].Name.L
 }
 
-// isColUnsigned returns true if the partitioning key column is unsigned.
-func isColUnsigned(cols []*model.ColumnInfo, pi *model.PartitionInfo) bool {
-	for _, col := range cols {
-		isUnsigned := mysql.HasUnsignedFlag(col.GetFlag())
-		if isUnsigned && strings.Contains(strings.ToLower(pi.Expr), col.Name.L) {
-			return true
-		}
+func isPartExprUnsigned(tbInfo *model.TableInfo) bool {
```

Comment:
I don't know whether it can be reused. If it can, It can be written like this.

```
func (t *TableInfo) IsPartExprUnsigned() bool {
```

Reply:
This is a bad idea.
`model` should not depend on `expression` package!

Reply:
Thank you @hawkingrei and @tiancaiamao for your comments, I assume this can be seen as resolved? Meaning the suggested helper function is OK and should not be a member function of TableInfo?
Any other review comments for this PR?